demo-projects/meetup/site/components/EventItems/EventItem.js

```diff
@@ -0,0 +1,78 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+
+import { Link } from '../../../routes';
+import { H2 } from '../../primitives/Typography';
+import { colors, gridSize, shadows } from '../../theme';
+import { isInFuture, formatPastDate, formatFutureDate } from '../../helpers';
+
+const EventItem = ({ id, name, startTime, description, talks, themeColor, ...props }) => {
+  const prettyDate = isInFuture(startTime)
+    ? formatFutureDate(startTime)
+    : formatPastDate(startTime);
+
+  // Breakpoints for the events grid
+  const breakpoints = [670, 890];
```

Comment:
Should these numbers live in `demo-projects/meetup/site/theme.js` instead?

Reply:
Probably! It was a quick implementation and nothing yet has media queries - we are in a very rapid _throw things at the wall_ mode and clean up as we go, but this is definitely something that should live in the theme file.